java/mtsj/pom.xml

```diff
@@ -155,13 +169,15 @@
       </plugin>
       <plugin>
         <artifactId>maven-surefire-plugin</artifactId>
-        <version>2.22.0</version>
+        <version>3.0.0-M4</version>
         <configuration>
-          <excludedGroups>${devon.test.excluded.groups}</excludedGroups>
```

Comment:
Why was this feature removed? This is intended by devon4j.

Reply:
As surefire was not working, I just copy a configuration that works and removed the previous one.